node/src/command.rs

```diff
@@ -38,7 +38,7 @@ use std::{io::Write, net::SocketAddr};
 fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
 	Ok(match id {
 		"dev" => Box::new(chain_spec::parachain_development_config(para_id)?),
-		"" => Box::new(chain_spec::roccocco_parachain_config(para_id)?),
+		"testnet" => Box::new(chain_spec::testnet_parachain_config(para_id)?),
```

Comment:
is this change necessary? you have to change local startup procedure as well and polkadot launch config files

Reply:
I think it's better if you have to specify a chain. You will know what you are running without looking into code to find out what chain is under `""`.  Polkadot launch running `--chain=local`. It should not be affected by this change. I can revert it back to `""` if we want

Reply:
ok, it is not issue for polkadot-launch afterall